services/src/main/java/org/keycloak/protocol/oidc/par/ParValidationService.java

```diff
@@ -0,0 +1,147 @@
+package org.keycloak.protocol.oidc.par;
+
+import org.jboss.resteasy.spi.HttpRequest;
+import org.keycloak.OAuth2Constants;
+import org.keycloak.OAuthErrorException;
+import org.keycloak.events.Details;
+import org.keycloak.events.Errors;
+import org.keycloak.events.EventBuilder;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.protocol.oidc.OIDCLoginProtocol;
+import org.keycloak.protocol.oidc.TokenManager;
+import org.keycloak.protocol.oidc.endpoints.request.AuthorizationEndpointRequest;
+import org.keycloak.protocol.oidc.endpoints.request.AuthorizationEndpointRequestParserProcessor;
+import org.keycloak.protocol.oidc.utils.OIDCResponseMode;
+import org.keycloak.protocol.oidc.utils.OIDCResponseType;
+import org.keycloak.protocol.oidc.utils.RedirectUtils;
+import org.keycloak.representations.idm.OAuth2ErrorRepresentation;
+import org.keycloak.services.ErrorResponseException;
+import org.keycloak.services.ServicesLogger;
+import org.keycloak.util.TokenUtil;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static org.keycloak.protocol.oidc.OIDCLoginProtocol.REQUEST_URI_PARAM;
+
+public class ParValidationService {
```

Comment:
I think this class is redundant and probably can be removed entirely from this PR?

Reply:
@mposolda Yes, it is not used elsewhere so that I will remove it.

Reply:
Thanks